server/service/live_queries.go

```diff
@@ -160,8 +223,16 @@ func (svc *Service) RunLiveQueryDeadline(
 		wg.Add(1)
 		go func() {
 			defer wg.Done()
-			campaign, err := svc.NewDistributedQueryCampaign(ctx, "", &queryID, fleet.HostTargets{HostIDs: hostIDs})
+			queryIDPtr := &queryID
+			queryString := ""
+			// 0 is a special ID that indicates we should use raw SQL query instead
```

Comment:
Are we sure 0 not used as legit query id?

Reply:
Yes. https://dev.mysql.com/doc/refman/8.0/en/example-auto-increment.html

> To start with an AUTO_INCREMENT value other than 1, set that value with [CREATE TABLE](https://dev.mysql.com/doc/refman/8.0/en/create-table.html) or [ALTER TABLE](https://dev.mysql.com/doc/refman/8.0/en/alter-table.html), like this:

And we are not altering or setting this for the `queries` table.